### DISCUSSION TITLE
Bun support

### DISCUSSION THREAD

**Discussion by ntnp** [spatie/file-system-watcher#23] [Ideas]
Hey.

My team is dropping node in favor of bun. We'd like to continue using the package, but the JS runtime is hardcoded to node.
```php
protected function getWatchProcess(): Process
{
    $command = [
        (new ExecutableFinder)->find('node'),
```

As long as the executable is findable by the OS, simply changing it to `bun` works.
Any chance we get a config to change the runtime?

PS:
The caveat is that bun isn't installed globally by default, so it needs to be linked as a global executable by the user (e.g. in the Dockerfile):

```Dockerfile
RUN ln -s ~/.bun/bin/bun /usr/local/bin/bun
```

**Comment by freekmurze** [comment]
Feel free to PR the necessary changes to make it easy to use bun.
